sensor/common/scan/scan_test.go

```diff
@@ -339,8 +350,65 @@ func (suite *scanTestSuite) TestEnrichNoRegistriesFailure() {
 	suite.Require().ErrorContains(err, "unable to create no auth registry")
 }
 
+func (suite *scanTestSuite) TestGetRegistries() {
```

Comment:
nit: I think you can just do the `setup` stuff in a `suite *scanTestSuite Setup()` method, and you can split all the inner tests (`suite.Run`) in separate, similarly named test methods

Reply:
I had that initially, but the testify suite was not playing nicely with the `vscode` debugger when running individual tests, to speed things along I removed the 'magic' and used the stdlib.